docs/examples/basic_tracer/README.rst

```diff
@@ -18,6 +18,7 @@ Installation
 
     pip install opentelemetry-api
     pip install opentelemetry-sdk
+    pip install opentelemetry-tools-google-cloud
```

Comment:
Alternatively, we could remove the code for GCP resource detection and just have OTEL resource detection from env `OTEL_RESOURCE_ATTRIBUTES`. We might want to add a note that vendor specific resource detection extensions live outside the open-telemetry and should be installed as a separate dependencies.

Reply:
I would prefer this approach. We don't want to add vendor specific stuff for such a basic example.